### PR TITLE
Dynamic volume control update after Dash

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Now that the blog post has been released, this documentation need to be updated.
Title and introduction have been changed.

### Motivation
Dash announcement

### Preview link
https://docs-staging.datadoghq.com/nils/logging-without-limits/logs/dynamic_volume_control/

### CODE REVIEW

**Comment by NBParis** [comment]
@l0k0ms As discussed, there might be some more things to do to change the title in the menu as well. Thanks for your help on this :)

**Comment by l0k0ms** [comment]
@NBParis I pushed the changes to update all links + reference to this page + aliases, it's ready for Q/A

File: content/logs/live_tail.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ further_reading:
 
 The Live Tail feature gives you the ability to see all your log events in near real time from anywhere in your infrastructure. It displays logs as soon as they get out of the [Pipeline section][1] and before [their indexation][2] by Datadog, hence: 
 
-1. All logs ingested by Datadog are displayed. ([The dynamic volume control][2] is not applied.)
+1. All logs ingested by Datadog are displayed. ([It's the logging without limit][2])
```
Comment (cswatt) [inline content/logs/live_tail.md]: It's logging without limits.

File: content/logs/logging_without_limits.md [hunk 2]
```diff
@@ -12,30 +14,35 @@ further_reading:
...
 
 * Ingest all your log events without server side filtering
-* Get alerted when volumes grows unexpectedly
 * Process and enrich all of them
 * Live Tail over the whole infrastructure
 * Dynamically decide what to include or exclude from your indexes to control your costs
+* Get alerted when volumes grows unexpectedly over an index
+* Archive all enriched logs
+
+This flexibility is critical in some exceptional situations such as outages, when you can disable specific filters to send more data. The inverse is true as well; if you over consume because of a seasonal reason (Black Friday, Christmas, etc…) you can decide to selectively reduce some volume to avoid overages.
```
Comment (cswatt) [inline content/logs/logging_without_limits.md]: r/over consume/over-consume

**Pull request merged**